recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set version = "2.2.4" %}
+{% set version = "2.3.0" %}
 
 package:
   name: libgdal
   version: {{ version }}
 
 source:
   url: http://download.osgeo.org/gdal/{{ version }}/gdal-{{ version }}.tar.xz
-  sha256: 441eb1d1acb35238ca43a1a0a649493fc91fdcbab231d0747e9d462eea192278
+  sha256: 6f75e49aa30de140525ccb58688667efe3a2d770576feb7fbc91023b7f552aa2
   patches:
     # Fixes compilation problems with JPEG.
     # https://lists.osgeo.org/pipermail/gdal-dev/2016-January/043443.html
-    - disable_jpeg12.patch
+    # - disable_jpeg12.patch
```

Comment:
@gillins if I got the code changes correctly this patch can be dropped.

Reply:
Yes I think you are right, but you will have to add `JPEG12_SUPPORTED=OFF` or something to `bld.bat`. I think....

You will also have to skip Py27 on Windows.

I'm a bit tied up for the next week or so but will have a proper look after that if it still isn't working.

Reply:
I think you need to change `OPENJPEG_CFLAGS="-I%LIBRARY_INC%" ^` -> `OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.3" ^` in `bld.bat`. Older versions of `libgdal` appended the `openjpegX.X` part automatically but doesn't seem to happen now....